docs/application/dotnet/guides/nui/resources.md

```diff
@@ -3,14 +3,47 @@
 NUI provides several ways to handle resource images.
 This tutorial describes the use of Image resources in NUI.
 
-## Overview
-
-Resources in NUI can apply Images or three-dimensional Models.
-
-The common method to access these resources is through Visuals.
-Controls use these visuals to display what is required.
-
-See [ImageView tutorial](imageview.md).
+## Image handling methods
+
+The two basic methods of handling image resources in NUI are as follows:
+ 
+1. `Visuals` are the main building block for UI components and provide reusable rendering logic that is controlled using properties. For more information, see [Visuals tutorial](visuals.md) page:
+    ```
+    //Define unique visual index.
```

Comment:
There seems a period at the end in some comments and in few it is not there. 
Request to check for consistency